windows-vagrant.md

```diff
@@ -1,252 +1,296 @@
 
-![image alt text](images/windows-vagrant-image_3.png)
+**Third**, open a web browser on your computer and go to: [http://127.0.0.1:3030](http://127.0.0.1:3030).  You will be able to preview the application in your web browser and it will look like this.
 
-After your server finishes spinning up and the text that says User Ctrl-C to stop appears it means your server is running.
+![preview](images/preview.png)
 
-3. Open a web browser on your computer and go to: [http://127.0.0.1:3030](http://127.0.0.1:3030/).
+If you see this, it means you've finished setting up your application and can preview an application that is running.
 
-You will be able to preview the application in your web browser and it will look like the following:
+## Step 7: Stop your rails server
 
-![image alt text](images/windows-vagrant-image_4.png)
+The command you ran above started running a server that can be previewed in a web browser.  Before we finish, we should close the program we just opened.
 
-If you see this page, it means you've finished setting up your application and can preview an application that is running.
+**Find your web development terminal where you ran the `rails server` (rails s) command and hold CTRL+C.**
 
-## Step Seven: Stop Your Rails Server
+After you press hold `CTRL+C` on your keyboard, your terminal will give you a message that indicates your server has closed and return the prompt so new commands can be entered.
 
-The command you ran above started running a server that can be previewed in a web browser. Before we finish, we should close the program we just opened.
+> ^C- Gracefully stopping, waiting for requests to finish
+> === puma shutdown: 2018-10-31 23:35:32 +0000 ===
+> - Goodbye!
+> Exiting
 
-To stop your rails server, complete the following:
+This will prevent the webpage from loading in the future, but will allow you to enter new commands.
 
-1. Find your web development terminal where you ran the rails server (`rails s`) command and press **CTRL+C**.
+## Step 8: Learn about your vagrant coding environment
 
-2. After you press **CTRL+C** on your keyboard, your terminal will give you a message that indicates your server has closed and return the following prompt so new commands can be entered:
+By following these instructions in this guide you've setup a virtual machine with vagrant on your computer.  This will be your coding environment that you will be able to use for the remainder of the course.
 
-```
-^C- Gracefully stopping, waiting for requests to finish === puma shutdown: 2018-10-31 23:35:32 +0000 ===
-Goodbye! Exiting
-```
+Since you will be using this coding environment extensively in the future, understanding it will be important.
+
+**_[Visit the the Introduction to Vagrant](cheat-sheets/vagrant-intro-windows.md)_** to learn more about how to use this development environment that you just setup.
+
+## Step 9: Enable virtualization (optional)
```

Comment:
Optional is partially misleading here (maybe "if necessary" instead - because it won't be optional for some people).